c++/src/Writer.cc

```diff
@@ -56,6 +57,7 @@ namespace orc {
       enableIndex = true;
       bloomFilterFalsePositiveProb = 0.05;
       bloomFilterVersion = UTF8;
+      timezone = "GMT";
```

Comment:
The Java ORC writer uses local time zone by default. However, the C++ ORC writer uses GMT by default in this [fix](https://github.com/apache/orc/commit/8ea3592f6aeddb5e6930798e4ff95065c6177e00). I hard-coded it to GMT at that time to work around the insufficient support of timezone conversion on the C++ side. I'd suggest to leave some comments to explain this history.

Reply:
Done